zipkin-lens/src/reducers/trace.js

```diff
@@ -15,28 +15,28 @@ import * as types from '../constants/action-types';
 
 const initialState = {
   isLoading: false,
-  trace: [],
+  traceSummary: {},
 };
 
 const trace = (state = initialState, action) => {
   switch (action.type) {
-    case types.FETCH_TRACE_REQUEST:
+    case types.TRACE_LOAD_REQUEST:
       return {
         ...state,
         isLoading: true,
-        trace: [], /* Initialize trace */
+        traceSummary: {}, /* Initialize trace */
```

Comment:
If it available in the initialstate, do we need still initialize agian?

Reply:
Yes, I think so.
I think previous `traceSummary` state should be cleaned up before loading a new trace.